charts/rancher-turtles/templates/core-provider.yaml

```diff
@@ -68,4 +64,17 @@ data:
       - apiGroups: ["rke-machine.cattle.io"]
         resources: ["*"]
         verbs: ["*"]
+---
```

Comment:
What if a user would want to use his own secret, already created in the cluster?

Reply:
I will add a flag to use an existing secret. The default behavior would be to create the pre-configured secret but the user can pass this parameter to avoid this.

Reply:
The simplest way how to achieve the goal - to not create anything if name and namespace is not set.

Reply:
The initial idea was to use an opt-out strategy rather than opt-in, so the secret would be created by default and the values for these two features set to true. They would be able to explicitly turn them off.